mediacapture-streams/permission-helper.js

```diff
@@ -4,7 +4,7 @@
 async function setMediaPermission(status="granted", scope=["camera", "microphone"]) {
   try {
     for (let s of scope) {
-      await test_driver.set_permission({ name: s }, status, true);
```

Comment:
From w3c/permissions#387 I took it that WebKit does support this parameter (unlike Gecko and Blink). Would this cause any kind of regression?

Reply:
Nope. Should be fine.